src/quo2/components/wallet/network_routing/animation.cljs

```diff
@@ -0,0 +1,110 @@
+(ns quo2.components.wallet.network-routing.animation
+  (:require [react-native.reanimated :as reanimated]))
+
+(def ^:private slider-timing 300)
+
+(defn show-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 1 slider-timing))
+
+(defn hide-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 0 slider-timing))
+
+(defn increase-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 8 300)
+  (reanimated/animate height-shared-value 40 slider-timing))
+
+(defn decrease-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 4 300)
+  (reanimated/animate height-shared-value 32 slider-timing))
+
```

Comment:
A few comments here:

- The `300` duration is redundant, that's the default.
- It would be better if we define any numbers here in constants
- I think using `width-shared-value height-shared-value` is unnessary. We can just call it width and height

Reply:
Hi @OmarBasem 
thanks for your review!

> The 300 duration is redundant, that's the default.

I think it's still better to have the value set explicitly, it's easier to manipulate and also more clear when reading in the future.

> It would be better if we define any numbers here in constants

What is the constants namespace you are referring to? I think having the values related to the animation close to this component is better.
cc  @J-Son89 do we have a pattern to write constants or something?

> I think using width-shared-value height-shared-value is unnessary. We can just call it width and height

I don't like adding `shared-value` at the end of the vars :cry:  it creates a verbose var name, but I find it better because while passing  these shared values we must be careful (they are hooks) and also it helps to future devs to read and understand the code.

I'd like to decrease the name length of functions related to manipulate shared values, e.g. `reanimated/set-shared-value` and `reanimated/get-shared-value` are very verbose and they make the code harder to read, also I'm naming hook vars with the suffix `-shared-value`, so I'm having:

`(reanimated/set-shared-value width-shared-value new-value)`

which is big for just a `set`.

@briansztamfater @J-Son89 @OmarBasem  What do you think?
I was thinking in something like `@` or `get` and `set!`, not sure about the names yet.

Reply:
no opinions on the reanimated api. @ could be confusing unless we're using `reanimated/@` ? then maybe it's not so bad.

Reply:
as for the constants, I believe we have many and it depends on the scope. In this case these values are based on this specific component then sure it live close to the component, of course if it's related to some reanimated default setting this should be at a scope that reflects that best 👍

Reply:
> reanimated/@

Yes, all of them would be namespaced. I think as soon as more developers are involved in animations, they will want to make those calls shorter.

> however in this case I think @OmarBasem is referring to the fact that you have a few values as 300 and not a defined variable. 🤷‍♂️

Ok, got it.
I'm not naming vars not reused/related. I think giving a name to all vars doesn't make a difference, I mean:

```clojure
;; I give a name when a var is reused because they are related to the same animation

(def my-timer-reused 100)

(defn foo []
  (reanimated/animate ,,, my-timer-reused)) 
 
(defn bar []
  (reanimated/animate ,,, my-timer-reused))
  
;; Avoid giving a name when it is only used in a single function:
(defn baz []
  (reanimated/animate ,,, 200))
```